go.mod

```diff
@@ -7,10 +7,19 @@ go 1.22.0
 // as Moby is not going to move to containerd v2 anytime soon or fix these transient dependencies.
 // We should still move back to upstream in the future, and remove our copy.
 
+// FIXME:
+// hcsshim is pinned until the transitive dependency to containerd v1.7 is resolved
+// https://github.com/microsoft/hcsshim/pull/2293/files
+replace github.com/Microsoft/hcsshim => github.com/Microsoft/hcsshim v0.12.7
+
+// FIXME:
+// plugins are pinned until they restore support for golang 1.22
+replace github.com/containernetworking/plugins => github.com/containernetworking/plugins v1.5.1
+
```

Comment:
Why do we need these replaces

Reply:
To pin the dependencies to that specific version.

Reply:
Then can we just leave go.mod as-is?

Reply:
Sure, we can.

The suggestion here is about being more friendly and explicit about why these two deps cannot move right now.

Right now, there is probably 2, or maybe 3 people who know why.
For anyone else, trying to upgrade dependencies will fail in confusing ways (now and in the future), making it less approachable to touch go.mod for contributors.